api/test/integration/types.test.js

```diff
@@ -0,0 +1,34 @@
+const { describe, test } = require('mocha')
+const { expect } = require('chai')
+
+const { setupApi, runProcess, utf8ToUint8Array, getLastTokenId, getToken } = require('../helper/routes')
+
+const USER_ALICE_TOKEN = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'
+const METADATA_KEY_LENGTH = 32
+const METADATA_VALUE_LITERAL_LENGTH = 32
+
+describe('types', function () {
```

Comment:
knowing that we are using all method e.g. getToken, runProcess and so on it would be nice to get some coverage on that